### PR TITLE
ci: set it up

### PR DESCRIPTION
**Pull request by Thomaash**

### CODE REVIEW

**Comment by Thomaash** [comment]
@mojoaxel I need the same permissions for this repo as in vis-network to enable CircleCI (and all the other repos I plan to set up).

**Comment by mojoaxel** [comment]
I'll do it tomorrow! Thx!

**Comment by mojoaxel** [comment]
> I need the same permissions for this repo as in vis-network to enable CircleCI (and all the other repos I plan to set up).

I changed your role from `Maintain` to `Admin`

File: .prettierrc.json [hunk 1]
```diff
@@ -0,0 +1,8 @@
+{
+  "parser": "typescript",
+  "singleQuote": true,
+  "printWidth": 120,
```
Comment (mojoaxel) [inline .prettierrc.json]: please remove this config file. see https://github.com/Thomaash/vis-configs

Comment (Thomaash) [reply .prettierrc.json]: We have to have a config which enables TS. I deleted the rest.

**Pull request merged**